api_specs/v3/component_analyses_v3.yaml

```diff
@@ -0,0 +1,413 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: '3.1'
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Dependency Analytics Server"
+security:
+- tokenAuth: []
+paths:
+  /component-vulnerability-analyses/:
+    post:
+      tags:
+        - CA batch call
+      summary: Get vulnerability details about a software component and recommendations to change to a more secured version in a batched manner.
+      parameters:
+        - name: uuid
```

Comment:
Comment: although i dont see any use of uuid, we can keep it in case we need in future. Make sure you dont have any validations for this field in the server code.

Reply:
Yes.I have keep most of the headers in place considering any integration in the future.